RELEASE-NOTES.md

```diff
@@ -179,6 +181,29 @@ If the node successfully mines a block, you shall read log entries like the foll
 2017-12-08 16:42:03.524 [info] <0.847.0>@aec_conductor:handle_mined_block:774 Block mined: Height = 1; Hash = 02764f5478587e34c04428e1b985925ca87a35258324412cf4749cce8e568136
 ```
 
+#### Verify that node mines on same chain as the testnet
+Inspect the current top header in the testnet. Its hash and height:
+```
+curl http://31.13.248.103:3013/v1/top
+{"hash":"HcebFAby1g7uLgj5KEp3jBsFnKXYEthrP5+r5P2BE9Q=","height":...
+```
+
+Observe your node's top and wait for it to sync with the other nodes (it
```

Comment:
At this point in the steps, the node is not yet connected to testnet.
A more suitable location for this new section would be after "Verify that node connected to the testnet" (100 lines below or so). This may also allow simplifying the added text (and feel free to integrate section "Verify that node connected to the testnet").